docker/nginx/conf.d/server/server.api

```diff
@@ -21,6 +21,7 @@ client_max_body_size 128k;
 rewrite ^/portals /skynet/portals permanent;
 rewrite ^/stats /skynet/stats permanent;
 rewrite ^/skynet/blacklist /skynet/blocklist permanent;
+rewrite ^/docs(?:/(.*))?$ https://sdk.skynetlabs.com/$1 permanent;
```

Comment:
I understand the match (`(.*)`) but I don't understand this part: `?:/`.

This document helped me a bit: 
https://nginx.org/en/docs/http/ngx_http_rewrite_module.html#rewrite

Reply:
`?:` means that the outside group is non matching so we don't pollute the `$1` result with unnecessary match